src/pages/Swap/Swap.tsx

```diff
@@ -104,6 +105,30 @@ export function Swap() {
     destAmount,
   } = useSwapStateFunctions();
 
+  useEffect(() => {
+    if (!selectedFromToken && !selectedToToken) {
```

Comment:
This check alone does not quite work, tokens are not restored when I close and re-open extension.

### Prod build:

https://github.com/user-attachments/assets/11e866de-ffec-499f-8af0-28aa140a065c


### This branch:

https://github.com/user-attachments/assets/b4f6887e-925c-4b59-87f2-b14a2eb94d14